### ISSUE TITLE
Datepicker accessibility

### ISSUE THREAD

**Issue by PigeardSylvain** [ng-bootstrap/ng-bootstrap#3753]
Sorry for my bad english ...

1. Name of days are not well read by screen readers : 
For example screenreader read "mo" instead of "monday". This could be fixed adding an aria-label on column headers

![image](https://user-images.githubusercontent.com/13221271/83122595-2c011200-a0d4-11ea-90a8-cb9c94efed18.png)

2. Screen reader says nothing when using previous and next month buttons. It would be nice to ear the new selected date, for example : "april 2020".

This can be achived by :
- Adding a span at the end of the datepicker, [hidden on all devices except screen readers](https://getbootstrap.com/docs/4.5/utilities/screen-readers/).
- Adding aria-live attribute on this span
- Updating this span when a new day, month or year is selected by the user
Example : 
`<span class="sr-only" aria-live="polite" aria-atomic="true">april 2020</span>`

3. This span could be used has a label for the grid using aria-labelledby
Example : 
`<ngb-datepicker>`
`   ...`
`   <div role="grid" aria-labelledby="full-date">`
`      ...`
`   </div>`
`   <span id="full-date" class="sr-only" aria-live="polite" aria-atomic="true">monday april 6 2020</span>`
`</ngb-datepicker>`

4. Datepicker in a popup
![image](https://user-images.githubusercontent.com/13221271/83165833-5b347500-a10e-11ea-994f-d7334f8d62ca.png)
A label should be added on the button (for screen readers), example : 
`<button><span class="sr-only">choose date</span></button>`

5. Range selection
Some informations should be read to screen readers, using the span in sr-only (cf. 4).
Example : 
When user select the first date, the span could be updated with : "thuesday may 12 2020, select end date"
When user select the second date : "from thuesday 12 may 2020 from friday may 15 2020"

#3745

**Comment by gpolychronis** [comment]
@PigeardSylvain  Hello.
About point 3, you talk about selected date, right?

**Comment by PigeardSylvain** [comment]
I say that a span could be used to store the month and year, the same span could be used then to store the full date when the user select a day.

This span could also be used to label the grid using an aria attribute.
